server/routes/induction/common/personalInterestsController.ts

```diff
@@ -35,8 +35,9 @@ export default abstract class PersonalInterestsController extends InductionContr
 
 const toPersonalInterestsForm = (inductionDto: InductionDto): PersonalInterestsForm => {
   return {
-    personalInterests: inductionDto.personalSkillsAndInterests.interests.map(interest => interest.interestType),
-    personalInterestsOther: inductionDto.personalSkillsAndInterests.interests.find(
+    personalInterests:
+      inductionDto.personalSkillsAndInterests?.interests.map(interest => interest.interestType) || null,
```

Comment:
Is this right? I think I'd expect the OR clause to return an empty array rather than null? 🤔

Reply:
Oops, not sure why I put null in here! Fixed.